src/ya-comiste-react-native/src/entrypoints/ExploreMore.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import * as React from 'react';
+import { Text } from 'react-native';
+import { fbt } from 'fbt';
+
+// type Props = {|
+//   +componentId: string,
+// |};
```

Comment:
Could also be removed?

Or you could leave it `type Props = {||}` and export the component as `export default (ExploreMore: ComponentType<Props>);`